tests/generative/variable_strategies.py

```diff
@@ -114,13 +166,19 @@ def count(_type, _frame):
         return st.builds(AggregateByPatient.Count, any_frame())
 
     @st.composite
-    def is_null(draw, _type, frame):
-        type_ = draw(any_type())
-        return Function.IsNull(draw(series(type_, frame)))
+    def operation(draw, type_, frame, operator_func):
```

Comment:
I might be tempted to call this `binary_operation` or `binary_op` or something like that just because I tend to use "operation" to refer to _any_ query model node whereas this is specifically for the two-place ones.

Reply:
Thanks - I couldn't work out what to call it :)